lib/backend/info/technicianDetails.ts

```diff
@@ -0,0 +1,58 @@
+enum TechnicianType {
+    Electrician = "Electrician",
+    Plumber = "Plumber",
+    Carpenter = "Carpenter",
+    Painter = "Painter",
+    Mason = "Mason",
+    CCTV = "CCTV",
+    AC = "AC",
+    Other = "Other"
+}
+
+
+
+class TechnicianDetails{
+    worktype: TechnicianType;
+    responseTime?: number;
```

Comment:
remove optional from here... we can instead set default values for all of them when making it in builder.